src/core/Facemesh.tsx

```diff
@@ -41,17 +45,15 @@ export const Facemesh = React.forwardRef<FacemeshApi, FacemeshProps>(
   ) => {
     const outerRef = React.useRef<THREE.Group>(null)
     const meshRef = React.useRef<THREE.Mesh>(null)
-    const [geometry] = React.useState(() => new THREE.BufferGeometry().setIndex(FacemeshDatas.TRIANGULATION))
 
     const [sightDir] = React.useState(new THREE.Vector3())
     const [sightDirQuaternion] = React.useState(new THREE.Quaternion())
```

Comment:
This should be a callback.
const [sightDirQuaternion] = React.useState(() => new THREE.Quaternion())